backend/database.js

```diff
@@ -109,9 +110,9 @@ sbp('sbp/selectors/register', {
     await sbp('chelonia.db/set', namespaceKey(name), value)
     return { name, value }
   },
-  'backend/db/lookupName': async function (name: string): Promise<string | Error> {
+  'backend/db/lookupName': async function (name: string): Promise<string> {
     const value = await sbp('chelonia.db/get', namespaceKey(name))
-    return value || Boom.notFound()
+    return value
```

Comment:
If you're going to modify this you need to make sure to modify every place that calls `'backend/db/registerName'`, e.g. now the logic in `'backend/db/registerName'` needs to be updated.

Reply:
I believe I already did this

Reply:
AFAICT that code hasn't been touched....

Reply:
No, but I think it needed to. Updated now.